addons/calendar/models/calendar_event.py

```diff
@@ -328,7 +328,8 @@ def _compute_stop(self):
         for event in self:
             # Round the duration (in hours) to the minute to avoid weird situations where the event
             # stops at 4:19:59, later displayed as 4:19.
-            event.stop = event.start + timedelta(minutes=round((event.duration or 1.0) * 60))
+            if event.start:
+                event.stop = event.start + timedelta(minutes=round((event.duration or 1.0) * 60))
```

Comment:
You should have a complete compute method assigning a value in all cases, not just a subset (otherwise you will have cache access issues notably). Like
```
if event.start:
    event.stop = ...
else:
    event.stop = event.stop or False
if event.stop and event.allday:
    event.stop -= timedelta(seconds=1)
```

Reply:
@tde-banana-odoo  IMO, its not needed since it already have `default=fields.Date.today` there.

I tried my patch again and seems to be Working well on all 3 cases.